docs/modules/ROOT/pages/getting-started/deploy/nixos.adoc

```diff
@@ -66,9 +61,12 @@ Make sure not to lose the signing keys and do deploy them to
 the target machine!
 ====
 
-4.
+== 4. Activation
+
 Apply the configuration by running `nixos-rebuild switch`.
 
+== 5. xref:getting-started/index.adoc#repository-setup[Continue with setting up your first repository]
```

Comment:
This xref is visually indistinguishable from normal headers (which are also links)
I guess we can adapt the RepositorySetup snippet for this.
(Same for the other deployment methods)